BlogAPI.Persistence/Models/Comment.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace BlogAPI.Persistence.Entities
+{
+    public class Comment
+    {
+        public int Id { get; set; }
+        public string Author { get; set; }
```

Comment:
Are there gonna be users in the system? Or can a random person just leave comments with whatever name they put in? If there's gonna be concept of users in this system, it would be probably better to have some sort of `UserID`.

Reply:
Yea was thinking about adding registration and login later on, for now I think this will suffice to start off